script/breakupScript.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+import numpy
+import numpy as np
+import scipy as sp
```

Comment:
Don't think you are using this import?

Reply:
Yes, that's right. It is removed.

_Before I followed the "Design guideline": "Viel hilft viel"_ :)